packages/connect-wallet-modal/src/connectButtons/connectTally.tsx

```diff
@@ -2,7 +2,7 @@ import { FC, useCallback } from 'react';
 import { useConnectorTally } from '@reef-knot/web3-react';
 import { Tally } from '@reef-knot/wallets-icons/react';
 import { ConnectWalletProps } from './types';
-import { ConnectButton } from '../components';
+import { ConnectButton } from '../components/ConnectButton';
```

Comment:
Maybe export it from `../components` in index.ts? so we don't need that specific imports 🤔